test/controllers/thesis_controller_test.rb

```diff
@@ -11,4 +11,23 @@ class ThesisControllerTest < ActionDispatch::IntegrationTest
     get '/thesis/new'
     assert_response :success
   end
+
+  test 'redirect after successful submission' do
```

Comment:
We'll want a test for an unsuccessful submission. I recall a conversation about what was in scope for this ticket and what was not...but don't remember where that was. If that was determined out of scope (probably) we'll need a ticket for the failed submission retaining values and displaying appropriate feedback, etc.

Reply:
1. You can has.

Reply:
Thx. I think we've got what we need for this PR but since it took a few days to cover I want to check it again after the weekend to make sure before merging. Thx!